src/main.c

```diff
@@ -18,15 +19,9 @@
 #include "mixer.h"
 #include "rc.h"
 
-extern void SetSysClock(bool overclock);
-
-serialPort_t *Serial1;
-
 int main(void)
```

Comment:
It would be super nice to bring back the `init()` and `loop()` functions, so we can use them in the SIL (can't have two mains, so I can't call main() directly, and there is a bunch of stuff I have to do in between loops). I don't like being arduino-y though, because it makes me feel like a goof.  What do you think?